### PR TITLE
Add Python 3.9 to build

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

### CODE REVIEW

**Comment by gliptak** [comment]
/assign @roycaihw

Travis failure seems to be an intermittent unrelated to this PR

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-   py27, py3{5,6,7,8}
-   py27-functional, py3{5,6,7,8}-functional
+   py27, py3{5,6,7,8,9}
+   py27-functional, py3{5,6,7,8,9}-functional
```
Comment (roycaihw) [inline tox.ini:4]: could you also update the tests in https://github.com/kubernetes-client/python-base?

**Comment by roycaihw** [comment]
/lgtm
/approve

Test flake fixed in https://github.com/kubernetes-client/python/pull/1329

File: tox.ini (hunk 1 above)
Comment (gliptak) [reply tox.ini:4]: @roycaihw https://github.com/kubernetes-client/python-base/pull/217

**Comment by gliptak** [comment]
@roycaihw https://github.com/kubernetes-client/python-base/pull/217

File: tox.ini (hunk 1 above)
Comment (roycaihw) [reply tox.ini:4]: Thanks

**Pull request merged**